### DISCUSSION TITLE
how to create an image carrousel similar to Instagram Stories interface?

### DISCUSSION THREAD

**Discussion by yotamguttman** [webstudio-is/webstudio-community#180] [Q&A]
I'd like to make a design which has a box that shows images (or an image slider), that resembles of the interface of Instagram stories. this means that there's a slider with arrows that let you skip to the next photos but also an indicator on top that has bars equal to the number of photos on the slider, that progresses fir a few seconds before it automatically switches to the next one.
![image](https://github.com/user-attachments/assets/72a94b0d-282f-409d-bc3d-5b3e8ddba6d3)

how would you go about it? and is it possible to make it into a 'module' I could replicate and use repeatedly on different pages and on each page I'd feature different photos?

**Comment by kof** [comment]
https://swiperjs.com/ ca do it and we have a template for it in the marketplace (inside builder)
